shard.lock

```diff
@@ -5,6 +5,6 @@ shards:
     version: 0.5.1
 
   secrets:
-    git: https://github.com/HCLarsen/secrets.git
-    version: 0.1.0+git.commit.9840e2c79b45829e12020633c272eedffea98a81
+    git: https://github.com/hclarsen/secrets.git
```

Comment:
This change appears to be in error, as the first three letters are capitalized in the repo URL.